torch/_inductor/decomposition.py

```diff
@@ -394,6 +395,48 @@ def bernoulli(self, *, generator=None):
     return torch.rand_like(self, dtype=torch.float32) < self
 
 
+@register_decomposition([aten.fmin, prims.fmin])
```

Comment:
should this go to _decomp instead? Doesn't look inductor-specific. Also, should this just lower to `std::fmin`/`libdevice.min` instead of being a decomp?

Reply:
There is already a decomp for this that maps to `prim.fmin`, this would conflict with that one.